pkg/api/mlflow/dao/repositories/run.go

```diff
@@ -266,8 +266,8 @@ func getMinRowNum(runs []models.Run) models.RowNum {
 
 // renumberRows will update the runs.row_num field with the correct ordinal
 func (r RunRepository) renumberRows(tx *gorm.DB, startWith models.RowNum) error {
-	if startWith <= models.RowNum(0) {
-		return eris.Errorf("attempting to renumber with 0 or less row number value")
+	if startWith < models.RowNum(0) {
+		return eris.Errorf("attempting to renumber with less than 0 row number value")
```

Comment:
I know it was before, but please change `Errorf` to `New`

Reply:
I would also like to point out that Errorf is also used in 2 other places with just strings. 
```go
// pkg/api/mlflow/dao/repositories/run.go
// line 175
if len(runs) != len(ids) {
			return eris.Errorf("count of deleted runs does not match length of ids input (invalid run ID?)")
		}
		
// and
// pkg/api/mlflow/dao/repositories/experiment.go
// line 138
if len(experiments) != len(ids) {
			return eris.Errorf("count of deleted experiments does not match length of ids input (invalid experiment ID?)")
		}
```

Should I change these to `New` too

Reply:
yes please! nice catch!